release/aws/aws.go

```diff
@@ -0,0 +1,65 @@
+package aws
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+)
+
+var awsClient *AWS
+
+type AWS struct {
+	session *session.Session
+}
+
+func initializeClient() error {
+	if awsClient != nil {
+		panic("called initializeClient twice")
```

Comment:
if it were public, I would definitely return the error. In this case, I chose to panic because this is only called from `GetClient`, and if called twice, we either raced (which shouldn't be happening in these release scripts) or violated some internal invariant